src/analytics_desktop.cc

```diff
@@ -0,0 +1,358 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "analytics/windows/include/public/c/analytics.h" // C API
+#include "firebase/app.h"                                  // For firebase::App
+#include "firebase/analytics.h"                            // For common Parameter, LogEvent, etc.
+#include "firebase/variant.h"                              // For firebase::Variant (used by Item and LogEvent)
+#include "firebase/future.h"                               // For firebase::Future
```

Comment:
All of these includes should be in a subdirectory e.g. "analytics/src/include/firebase/app.h" etc.